app/views/layouts/application.html.haml

```diff
@@ -1,18 +1,17 @@
-!!!
 %html
   %head
+    = render 'shared/navbar'
```

Comment:
is this mean to be in the head section? All visible HTML should be in the body tag.